Pracs/prac_05/state_names.py

```diff
@@ -0,0 +1,19 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT" : "Northern Territory", "WA" : "Western Australia",
+            "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ")
```

Comment:
You should modify this input statement to allow the user to enter in capitals or lower case but still receive the same output. (e.g. currently entering qld would be invalid)

Reply:
Oh of course, dunno how I missed that, thanks so much! :)